bot/cogs/coding/coding.py

```diff
@@ -242,7 +260,13 @@ async def run(self, ctx: Context, language: str, *, code: str = "") -> t.Union[N
         await returned.add_reaction("🗑")
 
         def check(reaction: discord.Reaction, user: discord.Member) -> bool:
-            return user == ctx.author and str(reaction.emoji) == "🗑" and reaction.message.id == returned.id
+            return (
+                all(
+                    user == ctx.author,
+                    str(reaction.emoji) == "🗑",
+                    reaction.message.id == returned.id
+                )
+            )
```

Comment:
```suggestion
            return all(
                user == ctx.author,
                str(reaction.emoji) == "🗑",
                reaction.message.id == returned.id
            )
```
This isn't a huge formatting mistake, but this seems like a better option here